### PR TITLE
PDS2ISIS Docs Update

### LINKED ISSUE #3648: pds2isis interpreting zero DN values to NULL by default
ISIS Version 3.5.2 +

**Description**  
pds2isis appears to be interpreting all 0 DN values as NULL by default for 8 bit images

**How to reproduce**  
Select an 8 bit pds image with zero values. Do not select the "Option to convert pixel values to NULL". Run the application. The output cube will have 0 DN values replaced by NULL values. 

**Additional context**  
The image we used was the NAC images for DOY 253 in 2019. The mostly consists of 0 DN values with the exception of a column of values toward the right edge of the image. 

The expected behavior is to leave 0 DN values as 0 if the option to convert to NULL is not selected.

### PR DESCRIPTION
**Pull request by acpaquette**
Update to pds2isis docs regarding zero values in 8bit images

## Description

Adds a section in the PDS2ISIS docs regarding ingesting 8bit PDS image using PDS2ISIS. Where any image pixel with a value of zero is converted to an ISIS NULL since ISIS needs some value for NULL and 0 was chosen from 0 - 255.

## Related Issue
Closes #3648 

## Motivation and Context
The LRO team was using PDS2ISIS to simply observe the pixel values of 8-bit LRO images, expecting to see pixel values of zero in certain spots on the image. Since ISIS converts all zero values in 8-bit images to NULLs this resulted in all zero values in the LROC image to be NULL. This behavior has been documented and a processing solution has been found.

## How Has This Been Tested?
N/A

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (jessemapel) [inline isis/src/base/apps/pds2isis/pds2isis.xml]: You should also mention that 255 is also reserved for High Instrument Saturation.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (jessemapel) [inline isis/src/base/apps/pds2isis/pds2isis.xml]: I wouldn't say that we recommend it for all 8-bit images. I would simply add that if you are working with an 8-bit image that needs 0 or 255 values represented as actual valid pixels, then use stretch.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (scsides) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: Something like: "It should be noted if any incoming DN values are equal to one of the ISIS special pixel (LINK) values for the corresponding bit type, those pixels will be special in the output cube."

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (scsides) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: Agreed

**Review by scsides — CHANGES_REQUESTED** [review]

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (jessemapel) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: It would be nice to tell users exactly what those DNs are for each bit type. Do we have a link in our glossary that has the special pixel values in it?

Comment (acpaquette) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: We would need to update the glossary entry for special pixels. Someone would also need to actually find what those values are. I looked around for a bit (no pun intended) but couldn't find anything concrete.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (scsides) [inline isis/src/base/apps/pds2isis/pds2isis.xml]: Missing ","
Think we should not single out 8 bit here. Either explain the details of all bit types here or maybe explain in general terms of special pixels and make sure the links point to the docs that show what the values are for specials.

**Review by scsides — CHANGES_REQUESTED** [review]

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (jessemapel) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: Special Pixel values are defined here:

https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/base/objs/SpecialPixel/SpecialPixel.h

Comment (krlberry) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: There is also a wiki page defining special pixel values, but the link is broken for some reason: https://usgs-astrogeology.github.io/ISIS3/gh-pages/Special_Pixels.html. I'll see if I can dig up the original on redmine.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (jessemapel) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: It's worth mentioning 8-bit in particular here because it's where this is most likely to come up. Some blurb saying that all DNs that are special pixel values for the bit type will not be interpreted by ISIS as valid pixels, here's the special pixel values for each bit type, this is most commonly an issue for 8-bit images, and then the stretch work-around.

**Review by krlberry — CHANGES_REQUESTED** [review]
If you are going to include information about special pixels in general, could you add a link to this wiki page explaining special pixel values in ISIS? 
https://github.com/USGS-Astrogeology/ISIS3/wiki/SpecialPixels

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (krlberry) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: I migrated the original to https://github.com/USGS-Astrogeology/ISIS3/wiki/SpecialPixels. I think it would be useful to link to this.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (acpaquette) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: @scsides Where exactly is missing a ","? I just want to add it to the appropriate location first go around.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (ladoramkershner) [inline isis/src/base/apps/pds2isis/pds2isis.xml]: link should go to 'Special Pixels'

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (ladoramkershner) [inline isis/src/base/apps/pds2isis/pds2isis.xml]: Link should go to Special Pixels

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (ladoramkershner) [inline isis/src/base/apps/pds2isis/pds2isis.xml]: How do user know what these values are? It is in the page Kristin recently ported over to the github wiki, but do we link that here or update the Special Pixels area of the ISIS Glossary to provide the same information?

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (acpaquette) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: I've linked the wiki page in the docs. There was some discussion on gitter about this not being a great way to go about things but should work for now.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (acpaquette) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: It might be worth adding the pixel values table to the glossary at some point but right now I have just linked the github wiki in the application documentation.

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (scsides) [reply isis/src/base/apps/pds2isis/pds2isis.xml]: between "255 either"

File: isis/src/base/apps/pds2isis/pds2isis.xml
Comment (jessemapel) [inline isis/src/base/apps/pds2isis/pds2isis.xml]: most *commonly*

**Review by krlberry — APPROVED** [review]
:+1:

**Review by jessemapel — APPROVED** [review]
Looks good. Gonna wait for @scsides to re-review before merging

**Review by scsides — APPROVED** [review]

**Comment by jessemapel** [comment]
Gonna merge and kill the CI build on this because it's just docs.

**Pull request merged**
